lib/entities/quote/DutchQuote.ts

```diff
@@ -358,15 +358,17 @@ export abstract class DutchQuote<T extends DutchQuoteRequest> implements IQuote
   }
 
   // Returns the number of gas units extra required to execute this quote through UniswapX
-  static getGasAdjustment(classicQuote: ClassicQuote): BigNumber {
+  // if adjustmentBps is provided, it reduce the gas adjustment by that factor
+  static getGasAdjustment(classicQuote: ClassicQuote, adjustmentBps = 0): BigNumber {
```

Comment:
I wonder if it would be better to explicitly adjust the start/endAmounts by adjustmentBps rather than adjusting the gas overhead applied to those amounts, especially since the gas adjustment is relative to the gas returned by classic (so possibly not accurate) and so the result isn't necessarily as expected / deterministic?

Reply:
Good q - this is more of a product requirement, and it has to do with how we are choosing to model "spread' for UniswapX. We don't really know how much we want to offset the end/tart of the auction, but we can always adjust the buffer and make sure we're within some bounded range.

This is especially true for smaller trades amounts, where the buffer is a big portion of the traded amount.